lib/src/Base/Func/openturns/EnumerateFunctionImplementation.hxx

```diff
@@ -69,6 +69,9 @@ public:
   /** Basis size from total degree */
   virtual UnsignedInteger getBasisSizeFromTotalDegree(const UnsignedInteger maximumDegree) const;
 
+  /** The marginal enumerate function */
+  virtual EnumerateFunctionImplementation getMarginal(const Indices & indices) const;
```

Comment:
```suggestion
  virtual EnumerateFunction getMarginal(const Indices & indices) const;
```

Reply:
you should have a forward declaration of the `EnumerateFunction` class also

Reply:
Thank you very much for the detailed suggestions. Fixed at 8ba0bdcdc775a35dbdf8f2987f7701ee352fa488.